packages/sfc-playground/src/VersionSelect.vue

```diff
@@ -111,4 +111,8 @@ onMounted(() => {
   border-top: 6px solid #aaa;
   margin-left: 8px;
 }
+
+.versions .active a{
```

Comment:
Plz format here using Prettier

Reply:
Running `pnpm format` globally doesn't seem to work, only manually adding spaces after `a`.